README.md

```diff
@@ -1 +1,39 @@
 # configuration-template
+
+A template for writing a [Configuration].
+
+To use this template:
+1. Replace the name, metadata in crossplane.yaml to reflect your Configuration.
+2. Add dependencies in crossplane.yaml.
+3. Update the sample CompositeResourceDefinition and Composition in the `apis/`
+   directory to reflect the API you're designing for your platform, and optionally add more if needed.
+4. Add/Update examples in the `examples/` directory.
+5. Update this file, `README.md`, to be about your Configuration!
+
+To build a Configuration package, use the Crossplane CLI:
+```shell
+# Just builds the package. It will go through the repo and collect yaml files.
+# Furthermore it will by default check for examples in `./examples` and add them to the package.
+crossplane xpkg build
+
+# You can specify various options for the build command, if needed
+crossplane xpkg build --package-root=test-directory --output=test-package.xpkg --examples-root=/test-examples
```

Comment:
this longer form command has a couple issues:

* the `--output` flag doesn't exist, apparently it's `-o` or `--package-file` 🤔 
* the test-examples dir doesn't exist

To avoid folks copying/pasting this command and getting confused when it doesn't work, how about this form which will work while still showing them all the flags they can optionally use?
```
crossplane xpkg build --package-root=. --package-file=test-package.xpkg --examples-root=./examples
```